### DISCUSSION TITLE
v1.85.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#4895] [Announcements]
# v1.85.0

> [!warning]
> **Breaking Changes**
> The server and mobile app **must** be on the same version for the application to work correctly.

## Highlights

Welcome to release `v1.85.0` of Immich. This release includes many bug fixes across the app, server, and web and introduces new features, which you can find in the highlights below.

- Global activity in a shared album and activity on the mobile app and album’s option
- Shuffle slideshow: New option to randomize the order
- Notable fix: Bulk data fetching for initial sync on the mobile app to fix the issue with a very large gallery, causing the app to crash due to an out-of-memory error.

### Global activity

The activity feature that was added in the last release has now been integrated into the shared album view, where you can now see the likes and comments for all photos and videos in the shared album. You can also like and comment directly on the album itself. In the new album options modal, you can also toggle this feature on and off.

In this release, the activity functions have also been added to the mobile app.

Thanks @martabal  and @shenlong-tanwen for your tireless work on this feature

https://github.com/immich-app/immich/assets/27055614/fafec2b1-ba56-487f-a965-f8c5bbca5efc

### Shuffle slideshow

With the return of our web expert, @brighteyed, the slideshow now plays and advances with a shuffle option by default. You can toggle the option to advance the slide show incrementally according to the creation date. 

![image](https://github.com/immich-app/immich/assets/27055614/212d2e48-9972-4334-ab6c-07a8214461d6)

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### Server
* feat(server): better API error messages (for unhandled exceptions) by @jrasm91 in https://github.com/immich-app/immich/pull/4817
* feat(server): make is favorite optional on asset upload by @jrasm91 in https://github.com/immich-app/immich/pull/4865
* fix(server): show/set activity like per user by @jrasm91 in https://github.com/immich-app/immich/pull/4775
* fix(server): duplicate faces bug by @jrasm91 in https://github.com/immich-app/immich/pull/4844
* chore(server): set db relations for `getByIds` by @mertalev in https://github.com/immich-app/immich/pull/4855
* chore(server): compose updates by @jrasm91 in https://github.com/immich-app/immich/pull/4878

### Web
* feat(web): Album's options by @martabal in https://github.com/immich-app/immich/pull/4870
* feat(web): shuffle slideshow order by @jrasm91 in https://github.com/immich-app/immich/pull/4277
* feat(web): global activity by @martabal in https://github.com/immich-app/immich/pull/4796
* fix(web): scrollbar not showing year by @alextran1502 in https://github.com/immich-app/immich/pull/4782
* fix(web): timeline alignment by @jrasm91 in https://github.com/immich-app/immich/pull/4808
* fix(web): scrollbar year label visibility by @waclaw66 in https://github.com/immich-app/immich/pull/4820
* fix(web): unstacking issues by @brighteyed in https://github.com/immich-app/immich/pull/4792
* fix(web): show one face for the same person in the detail panel by @martabal in https://github.com/immich-app/immich/pull/4822
* fix(web): unstack after stack child selection by @brighteyed in https://github.com/immich-app/immich/pull/4834
* fix(web): update like status by @martabal in https://github.com/immich-app/immich/pull/4803
* fix(web): optimize deps by @jrasm91 in https://github.com/immich-app/immich/pull/4877

### Mobile
* feat(mobile): share assets from album by @waclaw66 in https://github.com/immich-app/immich/pull/4821
* feat(mobile): shared album activities by @shenlong-tanwen in https://github.com/immich-app/immich/pull/4833
* feat(mobile): shared album activity disable handling by @shenlong-tanwen in https://github.com/immich-app/immich/pull/4890
* feat(mobile)!: batched full/initial sync by @fyfrey in https://github.com/immich-app/immich/pull/4840
* fix(mobile): allow upload if local asset in selection by @shenlong-tanwen in https://github.com/immich-app/immich/pull/4815
* fix(mobile): immich app bar tap radius by @shenlong-tanwen in https://github.com/immich-app/immich/pull/4816
* fix(mobile): shows asset date time with original timezone by @waclaw66 in https://github.com/immich-app/immich/pull/4774
* fix(mobile): first stack list asset is now highlighted on view by @upsetdog in https://github.com/immich-app/immich/pull/4802
* fix(mobile): backup indicator wrong when only background backup is enabled by @fyfrey in https://github.com/immich-app/immich/pull/4842
* fix(mobile): do not show trashed assets in the album viewer page by @shenlong-tanwen in https://github.com/immich-app/immich/pull/4894
* chore(mobile): change version mismatch text by @Wingysam in https://github.com/immich-app/immich/pull/4831

### Others
* docs: remove read-only related content by @alextran1502 in https://github.com/immich-app/immich/pull/4781
* docs: python upload guide by @jrasm91 in https://github.com/immich-app/immich/pull/4867
* chore: changes to `docker compose` command. by @jesb1n in https://github.com/immich-app/immich/pull/4828
* chore(deps): bump stumpylog/image-cleaner-action from 0.3.0 to 0.4.0 by @dependabot in https://github.com/immich-app/immich/pull/4886
* chore: Update README.md by @Strubbl in https://github.com/immich-app/immich/pull/4892
* docs: update by @alextran1502 in https://github.com/immich-app/immich/pull/4893

## New Contributors
* @jesb1n made their first contribution in https://github.com/immich-app/immich/pull/4828
* @Strubbl made their first contribution in https://github.com/immich-app/immich/pull/4892

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.84.0...

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.85.0'>v1.85.0</a>.</em>

**Comment by DrSpaldo** [comment]
Has the iOS version been released?

**Reply by adambeck7** [reply]
Yes, it's available.

**Comment by mmcer** [comment]
Sorry, can someone tell me what's the difference with app-release.apk and app-arm64-v8a-release.apk, can i always install app-release.apk on my phone since apk file is more large?

Thanks

**Reply by joelis10** [reply]
I assume the different APKs are for the different phone CPU architectures

**Comment by maltokyo** [comment]
Regarding this breaking change:

The /server-info/stats endpoint has been changed to /server-info/statistics

... Is there any action to take when upgrading? It is not clear for me what to do
